App.js

```diff
@@ -1,110 +1,26 @@
-  const [toCurrency, onChangetoCurrency] = useState("");
-
-  const uppercaseWords = (str) =>
-    str.replace(/^(.)|\s+(.)/g, (c) => c.toUpperCase());
-
-  function setFromCurrency() {
-    const inputFromCountry = CountryAndCurrency.getCountriesBy(
-      "name",
-      uppercaseWords(fromCountry)
-    );
-
-    if (inputFromCountry.length === 1) {
-      onChangefromCountryFlag(inputFromCountry[0].currency.unicode);
-      onChangefromCurrency(inputFromCountry[0].currency.code);
-    }
-  }
-  useEffect(() => {
-    setFromCurrency();
-
-  });
-
-  function setToCurrency() {
-    const inputToCountry = CountryAndCurrency.getCountriesBy(
-      "name",
-      uppercaseWords(toCountry)
-    );
-
-    if (inputToCountry.length === 1) {
-      onChangetoCountryFlag(inputToCountry[0].currency.unicode);
-      onChangetoCurrency(inputToCountry[0].currency.code);
-    }
-  }
-  useEffect(() => {
-    setToCurrency();
-  });
 
-  // if you want just rates, use this URL:
-  // `https://v6.exchangerate-api.com/v6/${API_KEY}/latest/${fromCurrency}`;
-
-  // can be separated to 'helper.js' file later.
-  const fetchData = () =>
-    fetch(
-      `${API_URL}/${API_KEY}/pair/${fromCurrency}/${toCurrency}/${fromAmount}`
-    )
-      .then((res) => res.json())
-      .then((data) => {
-        console.log(data)
-        onChangetoAmount(data.conversion_result)
-        console.log(toAmount)
-      })
-      .catch((error) => console.error(error));
-
-  const pressHandler = () => {
-    console.log(
-      "i am a button and i just got clicked! and now I will fetch data"
-    );
-    return fetchData()
+  const image = {
+    uri: "https://images.unsplash.com/photo-1612540646801-4d5d183e6bc1?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80",
   };
 
   return (
     // <TailwindProvider utilities={utilities}>
     <View style={styles.container}>
       <ImageBackground source={image} resizeMode="cover" style={styles.image}>
         <StatusBar style="auto" />
-        <Text style={styles.header}>Currency Exchange</Text>
-        <Main
-          fromCountry={fromCountry}
-          setFromCountry={setFromCountry}
-          fromCountryFlag={fromCountryFlag}
-          onChangefromCountryFlag={onChangefromCountryFlag}
-          fromAmount={fromAmount}
-          onChangefromAmount={onChangefromAmount}
-          fromCurrency={fromCurrency}
-          onChangefromCurrency={onChangefromCurrency}
-          toCountry={toCountry}
-          setToCountry={setToCountry}
-          toCountryFlag={toCountryFlag}
-          onChangetoCountryFlag={onChangetoCountryFlag}
-          toAmount={toAmount}
-          onChangetoAmount={onChangetoAmount}
-          toCurrency={toCurrency}
-          onChangetoCurrency={onChangetoCurrency}
-        />
-        <FlatButton text="Exchange Currency!" onPress={pressHandler} />
+        <Text style={styles.header}> Currency Exchange</Text>
```

Comment:
Extra space before currency not needed

Reply:
That, for some reasons, if I don't add another space before currency, it doesn't align center... I will show you today why I did it! If you are okay with that, I don't mind either